peachjam/views/legislation.py

```diff
@@ -379,3 +380,78 @@ def get_child_documents(self):
         )
         # TODO: we're not guaranteed to get documents in the same language, here
         return docs
+
+
+class UnconstitutionalProvisionListView(LegislationListView):
```

Comment:
If we're apply permission checks, we should be consistent and apply them here, too.

I don't think we have a strong need to apply them.

Reply:
for consistency i've added the permissions to the view

Reply:
@longhotsummer applying perms actually makes it so anonymous users cannot view the listing page or the tab. is this what we want?